LanguageFeatures/Augmentation-libraries/type_inheritance_A02_t01.dart

```diff
@@ -2,23 +2,19 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion An augmenting declaration may have no type annotations for a
-/// return type, variable type, parameter type, or type parameter bound type. In
-/// the last case, that includes omitting the extends keyword. For a variable or
-/// parameter, a var keyword may replace the type.
+/// @assertion If the type annotation or type parameter bound is omitted in the
+/// augmenting declaration, it is inferred to be the same as the corresponding
+/// type annotation or type parameter bound in the declaration being augmented.
 ///
-/// When applying an augmenting declaration that contains a type annotation at
-/// one of these positions, to a definition to be augmented, it's a compile-time
-/// error if the type denoted by the augmenting declaration is not the same type
-/// as the type that the augmented definition has at the corresponding position.
-/// An augmenting declaration can omit type annotations, but if it doesn't, it
-/// must repeat the type from the augmented definition.
+/// If the type annotation or type parameter bound is not omitted, then it's a
+/// compile-time error if the type denoted by the augmenting declaration is not
+/// the same type as the type in the corresponding declaration being augmented.
 ///
 /// @description Check that it is a compile-time error if augmenting declaration
```

Comment:
Typo:
```suggestion
/// @description Check that it is a compile-time error if an augmenting declaration
```
Seems to occur in every test, certainly in several.

Reply:
Thanks! Fixed here and below.